tensorflow/core/kernels/non_max_suppression_op.h

```diff
@@ -31,8 +32,25 @@ struct NonMaxSuppression {
                   int max_output_size,
                   typename TTypes<int, 1>::Tensor selected_indices);
 };
-
 }  // namespace functor
+#if GOOGLE_CUDA
+extern const int kNmsBoxesPerTread;
+// d_sorted_boxes_float_ptr is a pointer to device memory float array
```

Comment:
Also please make these argument descriptions a list with proper indention for better readability.